presto-product-tests/src/main/resources/sql-tests/testcases/window_functions/rowsRangeRankFunctions.sql

```diff
@@ -0,0 +1,9 @@
+-- database: presto; groups: window;
```

Comment:
Eh, it's mostly just more coverage on rank/dense_rank. I put it in since I was playing with it, but it probably doesn't add much value. It'd probably be more valuable to add coverage for lead/lag.